spec/test_prof/before_all/adapters/active_record_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module TestProf
+  module BeforeAll
+    def self.configure
+    end
+  end
+end
+
+require "test_prof/before_all/adapters/active_record"
+
+describe "TestProf::BeforeAll::Adapters::ActiveRecord" do
```

Comment:
It would be better (and I think easier and more useful in the long term) to write an integration test with real Active Record multi-db setup. What do you think?

Reply:
I tried it first but I ran into issues... but now I have a working setup, I can give it another shot.

Reply:
I struggled with the AR setup. I am now using an environment variable `MULTI_DB`, if true, an additional table `comments` is added in a separate sqlite database. 

If it is an OK solution, should we then run all variations in `rspec.yml` for single and multiple db setup? WDYT?

Alternative setup:
- Duplicate the AR models setup: 1 for Rails 5 (single db)and 1 for Rails 6 and up (multiple db)

Reply:
We don't need all the variations; let's run tests with `MULTI_DB=true` for Rails 7/master and `MULTI_DB=false` otherwise.

Reply:
Sure, changed it